File: sdk/src/test/java/com/uber/sdk/android/rides/RobolectricTestBase.java
package com.uber.sdk.android.rides;

import org.junit.runner.RunWith;
import org.robolectric.RobolectricGradleTestRunner;
import org.robolectric.annotation.Config;

/**
 * Base test class to remove use of annotations on all test classes.
 */
@RunWith(RobolectricGradleTestRunner.class)
@Config(constants = BuildConfig.class, sdk = 21)
public abstract class RobolectricTestBase {
}

File: authentication/src/test/kotlin/com/uber/sdk2/auth/RobolectricTestBase.kt
package com.uber.sdk2.auth

import org.junit.Rule
import org.junit.runner.RunWith
import org.mockito.junit.MockitoJUnit
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(sdk = [26])
abstract class RobolectricTestBase {
}
